mesh_aws_client/mesh_check_send_parameters_application.py

```diff
@@ -40,7 +43,7 @@ def initialise(self):
             self.system_config.get("CHUNK_SIZE", MeshCommon.DEFAULT_CHUNK_SIZE)
         )
         # TODO figure out better way to do this:
-        self.my_step_function_name = f"{self.environment}-mesh-send-message"
+        self.my_step_function_name = f"{self.environment}-send-message"
```

Comment:
You've renamed mesh-send-message to send-message. In list-recon, that might clash with email-send-message? Or other suppliers might have a send-message step function not for mesh?

Reply:
The given environment name is suffixed with "-mesh" here:
https://github.com/NHSDigital/spine-core-aws-common/pull/31/files#diff-e270f4f05d8454c15d5f8d2f7396a1afee4a4934cc34bb25c771ce96c8e45634R2

This ensures consistent naming and then it is passed into the lambdas on the "Environment" env var

Reply:
Also easiest to change this to get the step function name to get it from os.environ  and set that using terraform

Reply:
Ah. But then the parameters become /{environment}-mesh/mesh/mailboxes....
Please don't do that.

Reply:
You pass into all the lambdas an environment name that isn't actually the environment name. It would be better to pass in, something like:
  environment {
    variables = {
      StepFunctionName = ${local.name}-get-messages
      Environment = ${var.name_prefix}
    }
  }

Reply:
Or better yet, calculate the step function names in locals.tf and pass them through to  the lambdas and step functions